src/client/dfs/dfs_internal.c

```diff
@@ -259,6 +275,40 @@ dfs_hdl_insert(const char *str, int type, daos_handle_t *oh, struct dfs_mnt_hdls
 	return rc;
 }
 
+int
+dfs_hdl_cont_destroy(const char *pool, const char *cont, bool force)
+{
+	d_list_t		*rlink = NULL;
+	struct dfs_mnt_hdls	*hdl;
+	char			cont_pool[DAOS_PROP_LABEL_MAX_LEN * 2 + 1];
+	daos_size_t		len;
+
+	D_ASSERT(pool);
+	len = snprintf(cont_pool, strlen(pool) + strlen(cont) + 2, "%s/%s", pool, cont);
+	D_ASSERT(len >= strlen(pool) + strlen(cont) + 1);
```

Comment:
Minor: "D_ASSERT(len == strlen(pool) + strlen(cont) + 1)"? I am fine with current form too.

Reply:
yes indeed. i have done that for the others, but missed this one here. although this check is only for coverity ;-)
i will get it in a follow on PR i am working on as to not have to repush this again since it has passed CI.